python/examples/example_constant_species/ExampleEquilibrationLogic.ipynb

```diff
@@ -0,0 +1,587 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# AMICI documentation example of the steady state solver logic\n",
+    "\n",
+    "This is an example to document the internal logic of the steady state solver, which is used in preequilibration and postequilibration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Steady states of dynamical system\n",
+    "\n",
+    "Not every dynamical system needs to run into a steady state. Instead, it may exhibit \n",
+    " * continuous growth, e.g., $$\\dot{x} = x, \\quad x_0 = 1$$\n",
+    " * a finite-time blow up, e.g., $$\\dot{x} = x^2, \\quad x_0 = 1$$\n",
+    " * oscillations, e.g., $$\\ddot{x} = -x, \\quad x_0 = 1$$\n",
+    " * chaotic beviour, e.g., the Lorentz attractor\n",
+    " \n",
+    "If the considered dynamical system has a steady state for positive times, then integrating the ODE long enough will bring equilibrate the system to this steady state. However, this may be computational more demanding than other approaches and may fail, if the maximum number of integration steps is exceeded before reaching the steady state.\n",
+    "\n",
+    "In general, Newton's method will find the steady state faster than forward simulation. However, it only onvereges if started close enough to the steady state. Moreover, it will not work, if the dynamical system has conserved quantities which are not removed: conserved qunatities will cause singularities in the Jacobian of the right hand side of the system, such that the linear problem within each step of Newton's method can not be solved."
```

Comment:
```suggestion
    "In general, Newton's method will find the steady state faster than forward simulation. However, it only converges if started close enough to the steady state. Moreover, it will not work, if the dynamical system has conserved quantities which are not removed: conserved quantities will cause singularities in the Jacobian of the right hand side of the system, such that the linear problem within each step of Newton's method can not be solved."
```